libs/api/domains/national-registry-x-road/src/lib/nationalRegistryKeysXRoad.resolver.ts

```diff
@@ -0,0 +1,34 @@
+import { UseGuards } from '@nestjs/common'
+import { Resolver, Query } from '@nestjs/graphql'
+import { ApiScope } from '@island.is/auth/scopes'
+import {
+  IdsAuthGuard,
+  IdsUserGuard,
+  Scopes,
+  ScopesGuard,
+} from '@island.is/auth-nest-tools'
+import { Audit } from '@island.is/nest/audit'
+
+import { NationalRegistryXRoadService } from './nationalRegistryXRoad.service'
+import { NationalRegistryReligion } from '../models/nationalRegistryReligion.model'
+
+@UseGuards(IdsAuthGuard, IdsUserGuard, ScopesGuard)
```

Comment:
Should not have both IdsAuthGuard and IdsUserGuard. Should generally use IdsUserGuard in our graphql api.

Reply:
In fact, let's skip all of the guards and the Scopes decorator since we're not returning user data in this resolver.